src/library/simplified/extension/external_referrer.js

```diff
@@ -0,0 +1,8 @@
+    b["external_referrer"] = '';
```

Comment:
i'd prefer that data types not be mixed here (initialize as string, set as boolean in logic). if we're going to have it be a boolean, it should either be null or default to a boolean. we should probably just leave it uninitialized, as both the referrer and domain should always have values, and if it doesn't, something's broken.